### ISSUE TITLE
Support hash-pinned dependencies

### ISSUE THREAD

**Issue by lengland** [meanmail-dev/requirements#25]
**Is your feature request related to a problem? Please describe.**
I have a project that is based on [Poetry](https://python-poetry.org/) dependency management, but we convert that to a requirements.txt file occasionally for compatibility with other systems. Using Poetry's  export appends sha256 hashes of the dependency in the requirements.txt. This is not highlighted well.

Example poetry output:
```
requests==2.22.0 \
    --hash=sha256:9cf5292fcd0f598c671cfc1e0d7d1a7f13bb8085e9a590f48c010551dc6c4b31 \
    --hash=sha256:11e007a8a2aa0323f5a921e9e6a2d7e4e67d9877e85773fba9ba6419025cbeb4
```

**Describe the solution you'd like**
I would like to see this not highlighted as an error
It would be even better if there was a level of syntax highlighting on the `--hash` part

**Additional context**
There is no limit to the amount of hashes added to a dependency. Eg, `markupsafe` has 33!

**Comment by meanmail** [comment]
Thanks for feedback. Support for '--hash' is planned for the next major version 2020.2

**Comment by meanmail** [comment]
Relates to #18

**Comment by meanmail** [comment]
Fixed in 2020.4. Available for download
